packages/components/src/components/Modal/index.stories.tsx

```diff
@@ -1,46 +1,50 @@
 import React from 'react';
 import { H2, P, Modal } from '../../index';
 import { storiesOf } from '@storybook/react';
+import { text, boolean } from '@storybook/addon-knobs';
 
-storiesOf('Modals', module)
-    .add(
-        'Default',
-        () => (
-            <Modal cancelText="Close" data-test="modal">
-                <H2>Ahoj kamaráde!</H2>
-                <P>Ježíš tě strašně miluje.</P>
-            </Modal>
-        ),
-        {
-            info: {
-                text: `
-            ~~~js
-            import { Modal } from 'trezor-ui-components';
-            ~~~
-            `,
-            },
-        }
-    )
-    .add(
-        'padding with cancel',
-        () => (
+storiesOf('Modals', module).add(
+    'Default',
+    () => {
+        const heading = text('heading', 'Ahoj kamaráde!');
+        const description = text('description', 'Description');
+        const children = text(
+            'children',
+            'Sláva Otci i Synu i Duchu svatému, jako byla na počátku, i nyní, i vždycky a na věky věků.'
+        );
+        const cancelable = boolean('cancelable', true);
+        const useFixedWidth = boolean('useFixedWidth', false);
+        const useFixedHeight = boolean('useFixedHeight', false);
+
+        // const iconOptions: any = {
+        //     None: null,
+        // };
+        // variables.ICONS.forEach((icon: string) => {
+        //     iconOptions[icon] = icon;
+        // });
+
+        // const icon = select('Icon', iconOptions, 'ARROW_DOWN');
+
+        return (
             <Modal
-                padding="50px 20px"
-                cancelable
-                cancelText="Zavři už tenhle trapnej modálek"
-                data-test="modal-with-cancel"
+                data-test="modal"
```

Comment:
This is probably not necessary. It does not hurt anybody but I can't imagine such test scenario where I'd be looking for a generic "modal" component

Reply:
ah, sry, it's tests ofc..

Reply:
that reminded me I didn't delete the test scenario for modal-with-cancel 👍